src/style.css

```diff
@@ -1,3 +1,62 @@
-body {
+html {
+  --header-height: 60px;
+  --text-hover-color: #00ACDC
+}
+
+* {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: var(--header-height);
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead
```suggestion
```